azafea/event_processors/endless/metrics/v3/migrations/b0f69709d3f6_add_request_model_metrics_v3.py

```diff
@@ -0,0 +1,64 @@
+# type: ignore
+
+"""Add Request model for metrics v3
+
+Revision ID: b0f69709d3f6
+Revises: a82a26763a82
+Create Date: 2021-07-20 16:03:57.931104
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "b0f69709d3f6"
+down_revision = "a82a26763a82"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        "metrics_request_v3",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("sha512", sa.Unicode(), nullable=False),
+        sa.Column("received_at", sa.DateTime(timezone=True), nullable=False),
+        sa.Column("absolute_timestamp", sa.BigInteger(), nullable=False),
+        sa.Column("relative_timestamp", sa.BigInteger(), nullable=False),
+        sa.Column("channel_id", sa.Integer(), nullable=True),
+        sa.ForeignKeyConstraint(
+            ["channel_id"],
+            ["channel_v3.id"],
+            name=op.f("fk_metrics_request_v3_channel_id_channel_v3"),
+        ),
+        sa.PrimaryKeyConstraint("id", name=op.f("pk_metrics_request_v3")),
+        sa.UniqueConstraint("sha512", name=op.f("uq_metrics_request_v3_sha512")),
+    )
+    op.create_index(
+        op.f("ix_metrics_request_v3_channel_id"),
+        "metrics_request_v3",
+        ["channel_id"],
+        unique=False,
+    )
+    op.drop_column("invalid_aggregate_event_v3", "receveid_period_start")
+    op.drop_column("unknown_aggregate_event_v3", "receveid_period_start")
```

Comment:
Where did these columns come from? Why are they being removed in this migration? It looks like they were inadvertently present in the migrations which added these tables.

Assuming that's right: this fix should be in a separate migration, added in a separate commit, with a meaningful commit message.

Reply:
Yes, I saw this today, it was a column I had added and then removed in this table and I must have forgotten to re-generate the migrations after I removed them.

I'm going to put this in a separate commit, would you prefer I leave it in this PR or open another one for it?